nncf/quantization/algorithms/accuracy_control/ranker.py

```diff
@@ -98,8 +98,9 @@ def find_groups_of_quantizers_to_rank(self, quantized_model_graph: NNCFGraph) ->
             if x.metatype in self._algo_backend.get_quantizer_metatypes()
         ]
 
-        quantized_model_graph_without_shapeof = remove_shapeof_subgraphs(
-            deepcopy(quantized_model_graph), self._algo_backend.get_shapeof_metatypes()
+        quantized_model_graph_without_shapeof = deepcopy(quantized_model_graph)
+        remove_shapeof_subgraphs_inplace(
```

Comment:
I was not aware of the fact that this function is modifying original model, I made a mistake because of that and decided that it would be clearer to add `inplace` to function name. Should I remove it from the name?

Reply:
I reverted the function renaming